Deprecated Discord (#75)
nanos-world/docs: docs/core-concepts/scripting/discord-integration.md

local large_text = "Honk!"
local large_image = "nanos-world-full-world"

> Discord.SetActivity(state, details, largeImage, largeText)

Reviewer: ```suggestion
<MethodReference type="StaticClass" class_name="Discord" method="SetActivity" is_static show_class_name />
```

## Using your own Application ID

Using your own [Application ID](https://discord.com/developers/applications/) brings the advantage of being able to configure it the way you want, and even uploading your own images to be displayed in the Rich Presence.

![How it will look like](/img/docs/discord-integration-02.webp)

Initializing your own Application ID is extremely easy, just execute the following code on the Client side:

```lua title="Client/Index.lua"

> Discord.Initialize(clientId)

Reviewer: ```suggestion
<MethodReference type="StaticClass" class_name="Discord" method="Initialize" is_static show_class_name />
```

```

:::tip

> And you are done! Now you can use the `Discord.Initialize(clientId)`method!

Reviewer: ```suggestion
And you are done! Now you can use the <MethodReference type="StaticClass" class_name="Discord" method="SetActivity" is_static show_class_name /> method!
```
